scripts/reinstall-node-modules.js

```diff
@@ -0,0 +1,11 @@
+const {exec} = require('child_process');
+
+const isWindows = process.platform === 'win32';
+const command = `./tools/reinstall-node-modules${isWindows ? '.cmd' : ''}`;
+const executedBin = exec(command, error => {
+	if (error) {
+		console.error(`exec error: ${error}`);
+	}
+});
+executedBin.stdout.on('data', d => console.log(d));
```

Comment:
You need to add encoding option to exec or call `d.toString` here other wise you will print out a buffer.

And also nit: `d` -> `data`

Reply:
@cPhost take a look, it says default encoding is utf-8. https://nodejs.org/api/child_process.html#child_process_child_process_exec_command_options_callback

Reply:
it should be `utf8` if use got it inside the function this is not inside the function.